admin/build-doc

```diff
@@ -85,15 +85,20 @@ for bind in rados rbd cephfs rgw; do
     BUILD_DOC=1 \
         CFLAGS="-iquote$TOPDIR/src/include" \
         CPPFLAGS="-iquote$TOPDIR/src/include" \
-        LDFLAGS="-L$vdir/lib -Wl,--no-as-needed" \
+        LDFLAGS="-L$vdir/lib -Wl,--no-as-needed,-rpath=$vdir/lib" \
```

Comment:
Hmm, I think this rpath change will trigger some errors in the build service auto checks:
https://en.opensuse.org/openSUSE:Packaging_checks#Beware_of_Rpath

Reply:
@ddiss Thanks for the heads-up. I'll make sure we revert that downstream (it won't make any difference in functionality, since we do not run `admin/build-doc` downstream at all - that is the script upstream uses to build its web documentation).

Reply:
@smithfarm okay, thanks for the clarification. if we don't build it then I'm fine with it as is - IIRC the obs rpath checks are all based on build output, so the revert shouldn't be necessary.

Reply:
Yeah, the file is not packaged AFAIK. Thanks for noticing it, though.